conversions/conversions_case_test.go

```diff
@@ -0,0 +1,14 @@
+package conversions
+
+type romanToIntegerConversionTest struct {
+	input    string
+	expected int
+	name     string
+}
+
+var romanToIntegerTests = []romanToIntegerConversionTest{
```

Comment:
Move the testcases inside of the test file. We do not need it to separated in a standalone file.

Reply:
Should I rename the test file to `roman_to_integer_test.go` or you want all the tests in one file like the way it is now?